lib/wasm/WebAssemblyParser.js

```diff
@@ -22,6 +22,25 @@ const isMemoryImport = moduleImport => moduleImport.descr.type === "Memory";
  */
 const isTableImport = moduleImport => moduleImport.descr.type === "Table";
 
+const JS_COMPAT_TYPES = new Set(["i32", "u32", "f32"]);
```

Comment:
`u32` shouldn't be there, I believe it's an implementation details in `webassemblyjs` which showed up.

Also missing `f64`.